Meta/Websites/serenityos.org/index.html

```diff
@@ -52,6 +52,8 @@ <h1>SerenityOS</h1>
     <li><a href="happy/2nd/">Happy 2nd birthday! SerenityOS: The second year</a></li>
     <li><a href="happy/1st/">Happy 1st birthday! SerenityOS: From zero to HTML in a year</a></li>
     <li><a href="https://happy-serenityos.linus.dev/">Linus's ":^)" tracker</a></li>
+    <li><a href="https://serenity.linus.dev/">VNC-access to Serenity running on Linus's hardware</a></li>
```

Comment:
Please don't. I don't want to give any guarantees about uptime, and too many people using it makes the single instance unusable really quickly (even from a shared keyboard/mouse perspective). It's ok when people find it from Discord or the original announcement though, but I don't want it linked :)

Reply:
Ah, sorry, then I misunderstood you. Sure!